app/src/main/res/layout/activity_detail_movie.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center_horizontal"
+    android:layout_margin="@dimen/layout_margin"
+    android:orientation="vertical"
+    tools:context=".detail.DetailMovieActivity">
+
+    <LinearLayout
```

Comment:
What other approaches could be better to build a Responsive UI?

Reply:
I could used the ConstraintLayout or a RelativeLayout to do this. But for better performance and tooling support the documentation recommends the ConstraintLayout.